docs/docsite/rst/porting_guides/porting_guide_core_2.12.rst

```diff
@@ -47,6 +47,13 @@ Deprecated
         when: not teardown | bool
 
 
+Breaking Changes
+================
```

Comment:
I'd just put this under the `Plugins` section instead of introducing a new section. Things in the porting guide are already expected to be breaking changes/things that need to be ported. :)

I'll fix this. Everything else looks reasonable to me.